utils.js

```diff
@@ -397,7 +397,8 @@ function formatMessage(m) {
     timestampAbsolute: originalMessage.timestamp_absolute,
     timestampRelative: originalMessage.timestamp_relative,
     timestampDatetime: originalMessage.timestamp_datetime,
-    tags: originalMessage.tags
+    tags: originalMessage.tags,
+    reactions: originalMessage.reactions ? originalMessage.reactions : []
```

Comment:
It should be here:

![image](https://cloud.githubusercontent.com/assets/2154522/24465309/6ee80176-1461-11e7-85e0-921bb13f1ef9.png)

I just realized this actually corresponds to formatDeltaMessage, which I now notice you didn't add `reactions` to.  Is it possible to add it there?